frontend/src/app/components/wp-card-view/services/wp-card-view.service.ts

```diff
@@ -0,0 +1,37 @@
+import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/isolated-query-space";
+import {Injectable} from '@angular/core';
+import {WorkPackageResource} from "core-app/modules/hal/resources/work-package-resource";
+import {RenderedRow} from "core-components/wp-fast-table/builders/primary-render-pass";
+
+@Injectable()
+export class WorkPackageCardViewService {
+  public constructor(readonly querySpace:IsolatedQuerySpace) {
+  }
+
+  public classIdentifier(wp:WorkPackageResource) {
+    // The same class names are used for the proximity to the table representation.
+    return `wp-row-${wp.id}`;
+  }
+
+  public get renderedCards() {
+    return this.querySpace.rendered.getValueOr([]);
```

Comment:
Card view uses `RenderedRow` which is very table specific. I'll see if we can refactor this once this is merged to dev